audio/out/ao_pipewire.c

```diff
@@ -389,6 +540,33 @@ static int control(struct ao *ao, enum aocontrol cmd, void *arg)
     }
 }
 
+static void add_device_to_list(struct ao *ao, uint32_t id, const struct spa_dict *props, void *ctx)
+{
+    struct ao_device_list *list = ctx;
+    const char *name = spa_dict_lookup(props, PW_KEY_NODE_NAME);
+
+    if (!name)
+        return;
+
+    const char *description = spa_dict_lookup(props, PW_KEY_NODE_DESCRIPTION);
+
+    ao_device_list_add(list, ao, &(struct ao_device_desc){name, description});
+}
+
+static void list_devs(struct ao *ao, struct ao_device_list *list)
+{
+    // we are not using hotplug_{,un}init() because the AO core will only call
+    // the hotplug functions of a single AO. That will probably be ao_pulse.
+    if (pipewire_init_boilerplate(ao) < 0)
+        return;
+
+    ao_device_list_add(list, ao, &(struct ao_device_desc){});
```

Comment:
@t-8ch Is this needed? Seems redundant.

Reply:
This creates the device `'pipewire' (Default (pipewire))`.
It exists for the other AOs, so I also added it.